### PR TITLE
Updating the roles

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Roles

## Description

 Updated the roles linking them to CYF docs

## Rendered Pages

-----
[View rendered docs/finalproject/roles.md](https://github.com/CodeYourFuture/syllabus/blob/kfklein15-patch-3/docs/finalproject/roles.md)

### CODE REVIEW

File: docs/finalproject/roles.md [hunk 1]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager)
 
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Scrum Masters are expected to coordinate the team and make sure everyone knows what they are working on. They can provide support in
-
+Your main role is to coordinate the team and make sure everyone knows what they are working on. They can provide support in:
 - Scheduling meetings
-- Working with the team to create and update tasks on Trello
+- Working with the team to create and update tasks on Github Projects
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md:18]: 🎉 Great catch!

File: docs/finalproject/roles.md [hunk 2]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
 - Introducing and enforcing Agile practices (e.g. sprint planning, retros)
 
-### Technical Assistant
-
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Teaching assistant provide technical support to the trainees as they are developing their projects. They can provide support in
+### Technical Lead
+Full role description and commitment is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead)
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
Full [role description for Tech Lead](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead) (TL)
```
- Links must make sense out of context.  https://quail.readthedocs.io/en/latest/tests/aLinksMakeSenseOutOfContext.html
- Acronyms must be introduced once before they can be used
https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/meaning-located.html

File: docs/finalproject/roles.md [hunk 3]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
 
+The TL provide technical support to the trainees as they are developing their projects. They can provide support in:
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
The TL provides technical support to the trainees as they are developing their projects. They can provide support in:
```

Subject-verb agreement

File: docs/finalproject/roles.md [hunk 4]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
 - Architecture of the application
 - Helping to solve specific implementation problems
 - Helping to solve specific bugs or errors
 - Providing overview of the entire project
 
 ### UI/UX Designer
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
### User Interface/ User Experience (UI/UX) Designer
```
Acronyms should be introduced once before use. https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G102

File: docs/finalproject/roles.md [hunk 5]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
-
 _UI/UX Designers are spread across multiple teams_
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/ui-ux-designer).
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
Full [role description for UI/UX designer](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/ui-ux-designer).
```

Links must make sense out of context.

File: docs/finalproject/roles.md [hunk 6]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
...
 
-UI/UX'ers will provide their expertise in supporting the UI/UX trainees in each team. The can provide support in
-
+UI/UX'ers will provide their expertise in supporting the UI/UX trainees in each team. For example:
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
UI/UXers will provide their expertise in supporting the UI/UX trainees in each team. For example:
```

File: docs/finalproject/roles.md [hunk 7]
```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
 ## Volunteer Roles
 
 ### Product Manager // Scrum Master
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]:
```suggestion
### Product Manager / Scrum Master
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Minor SPAG changes and accessibility tweaks. Ta, @kfklein15 , I think that's all the reviews you sent me! Night night! ❤️

**Comment by kfklein15** [comment]
All committed, @SallyMcGrath

File: docs/finalproject/roles.md [hunk 8]
```diff
@@ -10,40 +10,34 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
 
 ## Volunteer Roles
 
-### Product Manager // Scrum Master
-
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Scrum Masters are expected to coordinate the team and make sure everyone knows what they are working on. They can provide support in
+### Product Manager / Scrum Master
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager)
```
Comment (SallyMcGrath) [inline docs/finalproject/roles.md]: Links must make sense out of context

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Links must make sense out of context. Every time you find yourself writing [here], stop yourself. ;)

File: docs/finalproject/roles.md (hunk 8 above)
Comment (kfklein15) [reply docs/finalproject/roles.md]: Strange, I accepted your change to it, not sure why it didn't do it.

Comment (kfklein15) [reply docs/finalproject/roles.md]: updated.

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**